src/Apps/Sell/Components/SubmissionLayout.tsx

```diff
@@ -45,9 +51,11 @@ export const SubmissionLayout: React.FC<SubmissionLayoutProps> = ({
             </Column>
           </GridColumns>
         ) : (
-          <FadeInBox maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={[2, 4]}>
-            {children}
-          </FadeInBox>
+          <Flex width={size} justifyContent="center">
```

Comment:
This should work as well without `useWindowSize()`.

```suggestion
          <Flex width="100vw" justifyContent="center">
```